### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by iefserge**
R = major

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "url": "git+https://github.com/micnews/deku-page-layout.git"
+  },
+  "bugs": {
+    "url": "https://github.com/micnews/deku-page-layout/issues"
+  },
+  "homepage": "https://github.com/micnews/deku-page-layout#readme",
+  "description": "Render page layout using Deku",
+  "dependencies": {
+    "escape-html": "^1.0.3",
+    "tsml": "^1.0.1",
```
Comment (kesla) [inline package.json]: isn't this a dev dependency?

**Comment by kesla** [comment]
Woohoooo! Great module!

File: package.json (hunk 1 above)
Comment (iefserge) [inline package.json]: yeah thanks

**Comment by ellell** [comment]
Yes, looking great!

**Pull request merged**